index.html

```diff
@@ -18,63 +18,73 @@
       <header class="header">
         <nav class="menu">
           <a href="#"><h2>My Logo</h2></a>
-          <a href="#"
+
+          <ul class="menu__links">
+            <li class="menu__links-item">Portofolio</li>
+            <li class="menu__links-item">About</li>
+            <li class="menu__links-item">Contact</li>
+          </ul>
+
+          <a class="humbeger" href="#"
             ><img src="/img/icons/Union.png" alt="toggle menu icon"
           /></a>
         </nav>
       </header>
 
       <section class="showcase">
-        <h1>I’m Joseph Glad to see you!</h1>
-        <p>
-          I’m a software developer! I can help you build a product , feature or
-          website Look through some of my work and experience! If you like what
-          you see and have a project you need coded, don’t hestiate to
-          contactnxn me.
-        </p>
-        <a href="#"><p class="follow">let's connect</p></a>
-        <ul class="social-media">
-          <li>
-            <a href="#"
-              ><img
-                src="/img/icons/twitter.svg"
-                alt="twitter"
-                title="follow me on twitter"
-            /></a>
-          </li>
-          <li>
-            <a href="#"
-              ><img
-                src="/img/icons/Linkedin_icon.svg"
-                alt="linkedin"
-                title="follow me on linkedin"
-            /></a>
-          </li>
-          <li>
-            <a href="#"
-              ><img
-                src="/img/icons/medium.svg"
-                alt="medium"
-                title="follow me on"
-            /></a>
-          </li>
-          <li>
-            <a href="#"
-              ><img
-                src="/img/icons/github.svg"
-                alt="github"
-                title="follow me on github"
-            /></a>
-          </li>
-          <li>
-            <a href="#"
-              ><img
-                src="/img/icons/angellist.svg"
-                alt="angellist"
-                title="follow me on"
-            /></a>
-          </li>
-        </ul>
+        <div class="showcase__presentation">
+          <h1>I’m Joseph Glad to see you!</h1>
```

Comment:
- [ ] GOOD JOB SO FAR 🏅. However some bugs need to be fixes. In you heading it is creating 3 lines which is not good. Please kindly make it 2 lines.

**Your design**
![image](https://user-images.githubusercontent.com/69011963/149137150-b630fe9d-a4a0-4261-acd6-c2dda3607638.png)

**Figma design**
![image](https://user-images.githubusercontent.com/69011963/149137192-e46c88f0-377f-452f-a260-f630b26636d1.png)